lambdas/tests/unit/handlers/test_lloyd_george_record_stitch_handler.py

```diff
@@ -164,86 +143,77 @@ def test_respond_500_throws_error_when_fail_to_upload_lloyd_george_file(
 MOCK_CLIENT_ERROR = ClientError(
     {"Error": {"Code": "500", "Message": "test error"}}, "testing"
 )
-
 MOCK_LG_DYNAMODB_RESPONSE_NO_RECORD = {"Items": [], "Count": 0}
-
-MOCK_LG_DYNAMODB_RESPONSE = {
-    "Items": [
-        {
-            "ID": "uuid_for_page_3",
-            "NhsNumber": "1234567890",
-            "FileLocation": f"s3://{MOCK_LG_BUCKET}/1234567890/uuid_for_page_3",
-            "FileName": "3of3_Lloyd_George_Record_[Joe Bloggs]_[1234567890]_[25-12-2019].pdf",
-            "Created": "2023-10-02T09:46:17.231923Z",
-        },
-        {
-            "ID": "uuid_for_page_1",
-            "NhsNumber": "1234567890",
-            "FileLocation": f"s3://{MOCK_LG_BUCKET}/1234567890/uuid_for_page_1",
-            "FileName": "1of3_Lloyd_George_Record_[Joe Bloggs]_[1234567890]_[25-12-2019].pdf",
-            "Created": "2023-10-02T09:46:17.231921Z",
-        },
-        {
-            "ID": "uuid_for_page_2",
-            "NhsNumber": "1234567890",
-            "FileLocation": f"s3://{MOCK_LG_BUCKET}/1234567890/uuid_for_page_2",
-            "FileName": "2of3_Lloyd_George_Record_[Joe Bloggs]_[1234567890]_[25-12-2019].pdf",
-            "Created": "2023-10-02T09:46:17.231922Z",
-        },
-    ]
-}
-
+MOCK_LLOYD_GEORGE_DOCUMENT_REFS = create_test_lloyd_george_doc_store_refs()
 MOCK_STITCHED_FILE = "filename_of_stitched_lg_in_local_storage.pdf"
```

Comment:
Could we move the constants and the fixtures to the top of the file for consistency across the test files?

Reply:
No problem :)   Just moved constants and the fixtures to the top for both the lambda's tests and service class tests.